QuizApp/App/Common/SceneDelegate.swift

```diff
@@ -1,12 +1,13 @@
 import UIKit
+import Factory
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
-    /// It is safe to force unwrap a coordinator here, as we want the app to crash
-    /// if something is not setup correctly.
-    private var coordinator: CoordinatorProtocol!
-    private var serviceFactory: ServiceFactory!
+    @Injected(Container.coordinator) private var coordinator
+    @Injected(Container.secureStorage) private var secureStorage
+    @Injected(Container.navigationController) private var navigationController
+    @Injected(UserContainer.userNetworkDataSource) private var userNetworkDataSource
```

Comment:
why are these injected?

Reply:
As per our call, I've only left the navigation controller to be injected as the coordinator needs it.

Reply:
Can you explain why you think navigationController needs to be **injected** to SceneDelegate if coordinator needs it?